hack/release.sh

```diff
@@ -91,7 +91,7 @@ ALLOW_OVERRIDE=${ALLOW_OVERRIDE:-}
 SKIP_BUILD=${SKIP_BUILD:-}
 SKIP_PUSH_LATEST=${SKIP_PUSH_LATEST:-}
 LINUX_ARCH=${LINUX_ARCH:-amd64 arm arm64 ppc64le s390x}
-WINDOWS_DISTROS=${WINDOWS_DISTROS:-ltsc2019 20H2}
+WINDOWS_DISTROS=${WINDOWS_DISTROS:-1809 20H2 lstc2022}
```

Comment:
typo, it's ltsc2022, isn't it?

Reply:
correct, I will move `WINDOWS_DISTROS` into Makefile in another PR, that would have the test converge in this repo's tests.